### DISCUSSION TITLE
Video upload error

### DISCUSSION THREAD

**Discussion by sousecode** [ReVanced/revanced-discussions#956] [General] [answered]
Hi! I can't upload videos through tiktok revanced. These are regional restrictions. In my region, i can't download videos in tiktok, but i can watch older videos without VPN. I want to suggest the idea of ​​removing the restriction for the mod revanced. Changing the ip doesn't help. And i can't change my profile photo. In other mods, it is possible to change the region and download video in tiktok. I love Revanced. I just offered an idea. Excuse me for my english, i use google translator.

**Comment by oSumAtrIX** [comment, marked as answer]
If it's region locked you will need a VPN.

**Reply by sousecode** [reply]
VPN does not help, these are restrictions in the application code. I'll shoot a video and show.

**Reply by oSumAtrIX** [reply]
Then it is account locked (through region):

![image](https://user-images.githubusercontent.com/13122796/228048295-24c6ec93-1099-4683-b22b-d8a709c3ed4e.png)
